### ISSUE TITLE
RCS1146: Does not get triggered

### ISSUE THREAD

**Issue by Debbus72** [JosefPihrt/Roslynator#223]
Found this bug in our code-base where we use/prefer (historically) null on the left-side.

Does trigger RCS1146:
```csharp
if (List != null && List.Count > 0)
```

Does not trigger RCS1146:
```csharp
if (null != List && List.Count > 0)
```

Btw, it does of course trigger RCS1098, and if we code-fix that, it then triggers RCS1146. But I think that it should trigger RCS1146 regardless where null is positioned.

**Comment by JosefPihrt** [comment]
Thanks for the report @Debbus72 !
